pkg/codegen/pcl/binder_schema.go

```diff
@@ -273,7 +307,13 @@ func (b *binder) loadReferencedPackageSchemas(n Node) error {
 			continue
 		}
 
-		pkg, err := b.options.packageCache.loadPackageSchema(b.options.loader, name, pkgOpts.version)
+		var pkg *packageSchema
+		var err error
```

Comment:
I'm wondering if this needs the same warning comment as we saw above about not using `getPackageSchema`. More generally, I'm not sure if `getPackageSchema` has a use/can be deprecated or not now...